package.json

```diff
@@ -29,6 +29,9 @@
     "test:scss": "stylelint \"source/sass/**/*.scss\" \"source/js/**/*.scss\" --syntax scss",
     "test:scss:styleguide:color": "stylelint \"source/sass/**/*.scss\" \"source/js/**/*.scss\" \"!source/sass/**/_colors.scss\" --syntax scss --config .stylelintrc-colors.js",
     "test:css": "stylelint \"network-api/networkapi/{,!(frontend)/**/}*.css\" --syntax scss",
+    "selenium": "run-p --race server test:site:selenium",
```

Comment:
@Pomax I got `ERROR: Task not found: "test:site:selenium"` from running `npm run selenium`.

Reply:
that should be `selenium:site:run`

Reply:
updated